src/modules/shared/components/Navigation/NavigationList/NavigationList.tsx

```diff
@@ -1,49 +1,94 @@
 import clsx from 'clsx';
-import Link from 'next/link';
-import { FC } from 'react';
+import { FC, ReactNode, useState } from 'react';
 
+import { Icon, IconLightNames, Overlay } from '@shared/components';
+
+import { NavigationItem } from '..';
 import styles from '../Navigation.module.scss';
+import { NavigationDropdown } from '../NavigationDropdown';
 
 import { NavigationListProps } from './NavigationList.types';
 
 const NavigationList: FC<NavigationListProps> = ({ items }) => {
+	const [openDropdown, setOpenDropdown] = useState('');
```

Comment:
```suggestion
	const [openDropdown, setOpenDropdown] = useState<string | undefined>(undefined);
```

Ben nooit echt fan geweest van 0-length strings als "empty"